[Merged by Bors] - chore(src/linear_algebra): linting

Vierkantor opened the pull request:
---

The `dual_pair` structure is a `Type` because one of its fields is `fintype`. Maybe we should refactor it to be a `Prop`?

robertylewis commented on src/linear_algebra/dual.lean:
```diff
@@ -118,23 +118,26 @@ by rw [← h.to_dual_total_left, h.total_repr]
 lemma to_dual_apply_right (i : ι) (v : V) : h.to_dual (B i) v = h.repr v i :=
 by rw [← h.to_dual_total_right, h.total_repr]
 
+/-- Apply `v` to the second argument of `to_dual : V →ₗ V →ₗ K`. -/
```
What does it mean to apply `v : V` to an argument of type `V`?

Vierkantor replied:
I'm having trouble finding a good phrasing. Maybe just write out the non-`Prop` contents:

```suggestion
/-- `h.to_dual_flip v` is the linear map sending `w` to `h.to_dual w v`. -/
```

robertylewis replied:
That sounds clearer to me!

Vierkantor committed:
Clear up docstring

Vierkantor committed:
chore(src/linear_algebra): linting

The `dual_pair` structure is a `Type` because one of its fields is `fintype`.
Maybe we should refactor it to be a `Prop`?
